apps/sumo-aws-apps/sumologic-app-utils/src/requirements.txt

```diff
@@ -0,0 +1,20 @@
+boto3==1.27.1
+botocore==1.30.1
+certifi==2023.5.7
+chardet==5.2.0
+charset-normalizer==3.3.1
```

Comment:
In general check if you can upgrade to latest version available and applicable today.